Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -589,6 +589,60 @@ private IEnumerator TestTouchableDistances(BaseNearInteractionTouchable touchabl
 
             yield return PlayModeTestUtilities.HideHand(handedness, inputSim);
         }
+
+
+        /// <summary>
+        /// Test the SetTouchableCollider(BoxCollider collider) method by changing the size of the new
+        /// box collider and rotating the object. This test is checking if the NearInteractionTouchable plane
+        /// is the same size as the box collider and in front of the object.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator NearInteractionTouchableSetTouchableCollider()
```

Comment:
And actually, what you could do is not add a new test method, but instead just remove the following two lines from  public IEnumerator NearInteractionTouchableVariant():

``
            touchable.SetLocalForward(touchNormal);
            touchable.SetBounds(new Vector2(0.5f, 0.5f));
``

That will actually test the bug fix (which is that adding touchable doesn't work out of the box).

Reply:
There are actually several places that do this -- it's probably good to remove all those lines since they are no longer needed.

Reply:
Just tried to test the method by removing those lines but, it does not pass because this test is moving the hands based on those specific bounds. 

The bounds for that test are (0.5, 0.5) but the actual size of the object's box collider is (1,1,1), so then calling SetTouchableCollider() would result bounds that are size (1,1).

It fails here:
```
                // No touch when moving outside the bounds
                yield return PlayModeTestUtilities.MoveHandFromTo(initialHandPosition + outOfBoundsOffset, objectPosition + outOfBoundsOffset, numSteps, ArticulatedHandPose.GestureId.Open, Handedness.Right, inputSim);
                yield return PlayModeTestUtilities.MoveHandFromTo(objectPosition + outOfBoundsOffset, rightPosition, numSteps, ArticulatedHandPose.GestureId.Open, Handedness.Right, inputSim);
                Assert.AreEqual(2, catcher.EventsStarted);
                Assert.AreEqual(2, catcher.EventsCompleted);
```

Reply:
The new test looks good to me.